build.gradle

```diff
@@ -283,7 +278,6 @@ project(':core') {
     testCompile project(':clients').sourceSets.test.output
     testCompile 'org.apache.hadoop:hadoop-minikdc:2.7.1'
     testRuntime "$slf4jlog4j"
-    zinc 'com.typesafe.zinc:zinc:0.3.7'
```

Comment:
Another reason to use it is if you want to use a newer Zinc than the one Gradle includes (although it seems like this may not work in some cases). Removing sounds good.